testing/generate.js

```diff
@@ -23,18 +26,35 @@ async function globalSetup() {
   await page.waitForSelector('.app-browser', {timeout: 180000}) // Wait three minutes for apps to sync
   await page.context().storageState({ path: 'state.json' })
   await browser.close()
+
+  fs.mkdirSync(resultsDir)
 }
 
 export default globalSetup
 `
 }
 
+function globalTeardown() {
+  return `// global-teardown.ts
+import * as fs from 'fs'
+
+const resultsDir = 'tmp'
```

Comment:
Since this is just going to be in the current working directory, I think we should pick a more descriptive name. My vote is for `test-results` or `results`.